packages/docusaurus-theme-classic/src/theme/NavbarItem/DocNavbarItem.tsx

```diff
@@ -45,14 +45,19 @@ export default function DocNavbarItem({
     [activeVersion, preferredVersion, latestVersion].filter(Boolean),
   );
   const doc = getDocInVersions(versions, docId);
+  const isCurrentDoc = activeDoc && activeDoc.sidebar === doc.sidebar;
```

Comment:
Name `isCurrentDoc ` is not really corrrect. I'd name this `isDocSidebarActive` instead